testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -75,6 +75,11 @@ Feature: Be able to list available channels and enable them
     And I should get "[I] RES-AS-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-as-8-updates-x86_64]"
     And I should get "[I] RES-CB-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-cb-8-updates-x86_64]"
 
+@susemanager
+  Scenario: Enable SLES15 SP4 SUMA client tools for creating bootstrap repositories
+    When I execute mgr-sync "add channel sle-manager-tools15-pool-x86_64-sp4"
```

Comment:
I hate this step where we put directly the command line inside the feature file...

Reply:
Personal preference but I agree, we could unify the commands to the style you mentioned below. Nevertheless this would be something for a future PR :wink: I just used what was already there.